src/sass/uploads/_uploadAreaDnD.scss

```diff
@@ -1,91 +1,98 @@
 .uploadAreaDnD {
   width: 40vw;
   height: 30vw;
-  border: 1px dashed $dblue;
+  border: 1px dashed $primary-blue;
   border-radius: 5px;
   padding: 15px;
   background-color: $blue-transparent;  
   opacity: 17%;
-  display:inline-block;
+  display: inline-block;
 }
+
 .dragging {
-  background-color: white;
-  border: 3px dashed $blue;
+  background-color: #fff;
+  border: 3px dashed $primary-blue;
 }
+
 .showValidation {
-  border: 1px dashed $danger;
+  border: 1px dashed $stanford-bright-red;
 }
+
 #fileUpload {
   display: none;
 }
+
 .fileArea {
-  height:80%;
+  height: 80%;
   width: 100%;
   text-align: center;
-  padding:5px;
-  margin:0;
-  overflow:scroll;
+  padding: 5px;
+  margin: 0;
+  overflow: scroll;
 }
+
 .oi-cloud-upload {
   font-size: 500%;
-  color: $blue-semi-transparent;
+  color: $blue-transparent;
```

Comment:
Will need to darken this, currently is a bit too transparent/light.

Reply:
I will address this issue in _SCSS_ for now.